README.md

```diff
@@ -2,16 +2,19 @@
 
 
 License : [![License](https://img.shields.io/badge/license-LGPL2-blue.svg)](https://github.com/analogdevicesinc/libiio/blob/master/LICENSE)
-Latest Stable Release : [![GitHub release](https://img.shields.io/github/release/analogdevicesinc/libiio.svg)](https://github.com/analogdevicesinc/libiio/releases/latest)
-Number of downloads :  [![Github All Releases](https://img.shields.io/github/downloads/analogdevicesinc/libiio/total.svg)](https://github.com/analogdevicesinc/libiio/releases/latest)
+Latest Release : [![GitHub release](https://img.shields.io/github/release/analogdevicesinc/libiio.svg)](https://github.com/analogdevicesinc/libiio/releases/latest)
+Downloads :  [![Github All Releases](https://img.shields.io/github/downloads/analogdevicesinc/libiio/total.svg)](https://github.com/analogdevicesinc/libiio/releases/latest)
+
+Forks :  [![GitHub forks](https://img.shields.io/github/forks/analogdevicesinc/libiio.svg?style=social&label=Fork&style=plastic)](https://github.com/analogdevicesinc/libiio/network)
```

Comment:
There's already the fork/watchers count on the top-right of the Github interface, this just surcharges the README, so I'd drop this.